code/game/objects/structures/tables_racks.dm

```diff
@@ -583,6 +589,73 @@
 	return ..()
 
 /*
+ * Glass
+ */
+
+/obj/structure/table/glass
+	name = "glass table"
+	desc = "A standard table with a fine glass finish."
+	icon_state = "glass_table"
+	parts = /obj/item/weapon/table_parts/glass
+
+
+/obj/structure/table/glass/attackby(obj/item/W as obj, mob/user as mob, params)
+	if (istype(W, /obj/item/weapon/grab) && get_dist(src,user)<2)
+		var/obj/item/weapon/grab/G = W
+		if (istype(G.affecting, /mob/living))
+			var/mob/living/M = G.affecting
+			if (G.state < GRAB_AGGRESSIVE)
+				if(user.a_intent == I_HURT)
+					if (prob(15))	M.Weaken(5)
+					M.apply_damage(15,def_zone = "head")
+					visible_message("<span class='warning'>[G.assailant] slams [G.affecting]'s face against \the [src]!</span>")
+					playsound(get_turf(src), 'sound/weapons/tablehit1.ogg', 50, 1)
+					playsound(src.loc, "shatter", 50, 1) //WRESTLEMANIA tax
+					new /obj/item/weapon/shard(src.loc)
+					new /obj/item/weapon/shard(src.loc)
+					qdel(src)
+				else
+					to_chat(user, "<span class='warning'>You need a better grip to do that!</span>")
+					return
+			else
+				G.affecting.forceMove(loc)
+				G.affecting.Weaken(5)
+				visible_message("<span class='warning'>[G.assailant] puts [G.affecting] on \the [src].</span>")
+			returnToPool(W)
+			return 
+	return ..()
+/obj/structure/table/glass/flip()
+
+
+	if(prob(70))
+		. = ..() 
+	else
+		budgemute = 1
+		playsound(src.loc, "shatter", 50, 1)
+		new /obj/item/weapon/shard(src.loc)
+		new /obj/item/weapon/shard(src.loc)
+		qdel(src)
+		usr.visible_message("<span class='warning'>[usr] flips \the [src] and it shatters!</span>")
+
+/obj/structure/table/glass/kick_act(mob/living/carbon/human/H)
+	..()
+	playsound(src.loc, "shatter", 50, 1)
+
+	if(!H) return
+	if(prob(70))
+		do_flip()
```

Comment:
Changing ``` . = ..() ``` to ``` do_flip() ``` resulted in every table shattering every time and tripling the ``` to_chat ``` output

``` 

Bryant Wood flips the glass table and it shatters!

Bryant Wood flips the glass table!

Bryant Wood flips the glass table!

Bryant Wood flips the glass table!

Bryant Wood flips the glass table!

Bryant Wood flips the glass table and it shatters!

Bryant Wood flips the glass table!

Bryant Wood flips the glass table!

```